spec/views/hq/partners/index_spec.rb

```diff
@@ -1,11 +1,27 @@
 require 'rails_helper'
+require 'will_paginate/array'
 
 RSpec.describe "hq/partners/index.html.erb", type: :view do
   context 'partners exist' do
+    let(:partners) do
+      (1..31).each_with_object([]) do |num, arr|
+        arr << FactoryGirl.build_stubbed(:partner)
+      end.paginate(:page => 2)
```

Comment:
you could pass in the :per_page value here (something like 2) and then could create much fewer records for you test